okhttp/src/main/java/okhttp3/Headers.java

```diff
@@ -275,6 +285,21 @@ static void checkValue(String value, String name) {
     }
   }
 
+  public interface Listener {
```

Comment:
Doesn't seem like it should public API

Reply:
It's used inside `Http2Stream` to forward received headers blocks.

Reply:
That's an internal class. This callback is unused by public API.

Reply:
We're trying to figure out where to put it for duplex calls, when users will be able to subscribe to headers updates.